persistence-modules/spring-data-redis/src/main/java/com/baeldung/spring/data/redis_ttl/entity/Subscriber.java

```diff
@@ -0,0 +1,37 @@
+package com.baeldung.spring.data.redis_ttl.entity;
+
+import java.io.Serializable;
+
+import org.springframework.data.annotation.Id;
+import javax.persistence.Entity;
+
+@Entity
+public class Subscriber implements Serializable{
+    private static final long serialVersionUID = 1L;
+    @Id private Long id;
```

Comment:
Let's put a line break after `@Id` and let's separate fields from each other with blank lines.